### PR TITLE
Blob fix

### PR DESCRIPTION
**Pull request by erinlimbo**
-Removed ImagesService
-Replaced with Blobstore url serving

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/BlobServe.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  /** The Blobstore Service. */
+  BlobstoreService blobstoreService = BlobstoreServiceFactory.getBlobstoreService();
+
+  /** Read the data from the datastore and write it into /meme-handler as json. */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    BlobKey blobKey = new BlobKey(request.getParameter("blob-key"));
+    blobstoreService.serve(blobKey, response);
+    }
```
Comment (laskdaf) [inline portfolio/src/main/java/com/google/sps/servlets/BlobServe.java]:
```suggestion
  }
```

File: portfolio/src/main/java/com/google/sps/servlets/BlobServe.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Serve a single blobkey.
+ */
+@WebServlet("/blob-serve-url")
+public class BlobServe extends HttpServlet {
```
Comment (laskdaf) [inline portfolio/src/main/java/com/google/sps/servlets/BlobServe.java:18]: Should this be BlobServlet?

**Review by laskdaf — APPROVED** [review]
LGTM. Please address comments before merging.

File: portfolio/src/main/java/com/google/sps/servlets/BlobServe.java (hunk 2 above)
Comment (erinlimbo) [reply portfolio/src/main/java/com/google/sps/servlets/BlobServe.java:18]: Nope I named my java file BlobServe

**Pull request merged**